templates/web/cyclinguk/tokens/confirm_problem.html

```diff
@@ -0,0 +1,36 @@
+[% pre_container_extra = BLOCK %]
```

Comment:
I can't really see a nicer way of doing this at present that doesn't involve littering the base templates with 'IF cobrand==cyclinguk', or moving all the "main content" parts out into separate templates that can be included, which isn't exactly ideal either. SO perhaps leave as is for now (though I think reusing next_steps unless there's a reason not to do that) and we can see if we can think of something better later.

Reply:
I'll leave as it is for now. I'm using currently using `next_steps` and got rid of the file I created. =)